### PR TITLE
add new parameter in the config to store a list of URIs that the MLR

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
auth is allowed to redirect to.

For AQS-PI and whatever other systems need to auth against MLR via a
browser.  Also upgraded some dependencies.

### CODE REVIEW

**Review by zmoore-usgs — APPROVED** [review]
Looks good to me!

**Review by SMH94 — APPROVED** [review]

**Comment by SMH94** [comment]
Nice tests!

File: src/main/java/gov/usgs/wma/mlrgateway/controller/BaseController.java [hunk 1]
```diff
@@ -26,14 +26,17 @@
...
 	
 	@Value("${notification.email.cc-list:}")
 	protected void setNotificationEmailCCListString(String notificationEmailCCListString) {
 		this.notificationEmailCCListString = notificationEmailCCListString;
 	}
-
-	@Value("${uiHost:}")
-	protected String uiDomainName;
+	
+	@Value("${allowedRedirectUris:}")
```
Comment (cschroed-usgs) [inline src/main/java/gov/usgs/wma/mlrgateway/controller/BaseController.java:36]: Untested:

It looks like the @Value annotation can convert CSV strings to Arrays of Strings.
https://www.baeldung.com/spring-inject-arrays-lists

Example...
```
@Value("${myListOfValues}")
private String[] valuesArray;
```


...or in this case something more like this?...

```
        protected String[] allowedRedirectUris;
        ...
	protected void setAllowedRedirectUriString(String[] allowedRedirectUris) {
		this.allowedRedirectUris = allowedRedirectUris;
	}
```

Would @Value's built-in CSV parsing work, or does parsing using StringUtils.split offer an advantage?

**Review by cschroed-usgs — APPROVED** [review]
Looks good to merge!

File: src/main/java/gov/usgs/wma/mlrgateway/controller/BaseController.java (hunk 1 above)
Comment (cschroed-usgs) [reply src/main/java/gov/usgs/wma/mlrgateway/controller/BaseController.java:36]: (this is not a blocker)

Comment (kmschoep-usgs) [reply src/main/java/gov/usgs/wma/mlrgateway/controller/BaseController.java:36]: That's cool, and should work fine.

**Pull request merged**